ingest-mapper/src/main/scala/uk/gov/nationalarchives/ingestmapper/MetadataService.scala

```diff
@@ -32,6 +32,15 @@ class MetadataService(s3: DAS3Client[IO]) {
 
   def parseBagInfoJson(input: Input): IO[List[Obj]] = parseFileFromS3(input, "bag-info.json", _.map(bagInfoJson => Obj.from(read(bagInfoJson).obj)))
 
+  private def addChildCountValues(metadataJson: List[Obj]): List[Obj] = {
```

Comment:
Maybe `addChildCountAttribute` or `addChildCountObj`?

Reply:
I'll go with `addChildCountAttributes` as there's more than one.

Reply:
Ahh, I see. I guess `addChildCountAttribute` could be interpreted two ways; what I meant was "add the childCount attribute (to each row)". The thing is `addChildCountAttributes` could also be interpreted a different way e.g. "add multiple childCount Attributes" to each row (childCount1, childCount2). 

Maybe `addChildCountToEachRow`? Otherwise, maybe your suggestion is better

Reply:
I think it's fine with `addChildCountAttributes` You can see from the code that it's adding one attribute for each row.